coralproject/talk
# Exclude deleted comments

## What does this PR do?

Adds a new input field `excludeDeleted` which allows some fields to exclude deleted comment data. This is used on the Admin to exclude deleted items from the moderation interface.

## How do I test this PR?

1. Create a user.
2. Create many comments, and replies to other user's comments.
3. Delete the user using the cli tool: `./bin/cli users delete <userID>`
4. Visit the Admin and notice that the comments from the deleted user do not appear and the comment count in the mod queues reflect the correct (undeleted) comment count.
5. Visit the embed stream and notice that the comments do appear as a tombstone.
